spec_plots/utils/specutils_hasp/plotspec.py

```diff
@@ -0,0 +1,347 @@
+"""
+.. module:: plotspec
+   :synopsis: Creates preview plots for the provided HASP spectrum.
+
+.. moduleauthor:: Scott W. Fleming <fleming@stsci.edu>
```

Comment:
This script is based on the plotspec.py from cos/stis, but was updated significantly, so could update author to Swaters.

Reply:
Updated author to Swaters.